core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/scheduling/DummyJob.java

```diff
@@ -22,7 +22,7 @@
 
 public class DummyJob extends ScheduledJob
 {
-    private volatile boolean hasRun = false;
+    volatile boolean hasRun = false;
```

Comment:
Nit: protected?

Feel free to leave like this as well.

Reply:
Let's stay with package private. More restrictive.